src/main/java/user/deletereview/DeleteReviewInteractor.java

```diff
@@ -21,6 +27,11 @@ public DeleteReviewInteractor(IReviewDAO dataAccessReview, IInternshipDAO dataAc
         this.userDAO = userDAO;
     }
 
+    /** Deletes the review as specified by the inputs in requestModel from the review database.
+     * @param requestModel the request model
+     * @return a response model to be sent back to the client
+     */
+    @Override
```

Comment:
Maybe add an one-sentence explanation about the try catch exception

Reply:
Leo told me how to write this one because I was mentioning that stuff like the try/catch exception and I was saying too much. He said just to explain what the method does without going into the implementation. So as long as there is not an exception that can be thrown (because in this case it catches it and gives a message instead) then I shouldn't be writing anything about the try/catch.